spyglass/src/main/java/com/linkedin/android/spyglass/ui/MentionsEditText.java

```diff
@@ -69,6 +79,9 @@
  */
 public class MentionsEditText extends EditText implements TokenSource {
 
+    private static final String KEY_MENTION_SPAN = "mention_span";
+    private static final String KEY_MENTION_SPAN_START = "mention_span_start";
```

Comment:
My bad, these should be named `KEY_MENTION_SPANS` and `KEY_MENTION_SPAN_STARTS` since we're storing arrays here and not a single item (please also update the strings to "mention_spans" and "mention_span_starts" for consistency).